sway/commands/new_float.c

```diff
@@ -0,0 +1,7 @@
+#include "log.h"
+#include "sway/commands.h"
+
+struct cmd_results *cmd_new_float(int argc, char **argv) {
+	sway_log(SWAY_INFO, "Warning: new_float is deprecated");
```

Comment:
This should include instructions on how to fix it (e.g. "Warning: new_float is deprecated. Use floating_border instead.")